### PR TITLE
Update template image of image payload

### LINKED ISSUE #1115: [Demand] Update image of image-based payload
## Metadata

Field | Value
--- | ---
Previous Stage | ...
Figma Link | https://www.figma.com/file/4DmpRxZk4jDeqrfA5xrPCp/Dashboard-v3?node-id=985%3A0
Expected UX prototype due | 2020-W??-5
Actual UX prototype release | -
Expected engineering due | 2020-W??-5
Corresponding Pull Request | -
Completion Version | -

## Catalogue

- ...
- ...
- ...

### PR DESCRIPTION
**Pull request by guanbinrui**
close #1115

### CODE REVIEW

**Comment by Tedko** [comment]
@septs can you run img optimization for these as well?

**Comment by septs** [comment]
@guanbinrui your self can use imageoptim.com ?

**Comment by yisiliu** [comment]
> @guanbinrui your self can use https://imageoptim.com ?

502 Bad Gateway?

**Comment by septs** [comment]
> > @guanbinrui your self can use https://imageoptim.com ?
> 
> 502 Bad Gateway?

https://imageoptim.com/mac

**Comment by Tedko** [comment]
will ignore the img optimization and merge now. we can do img optimization later.

**Comment by Jack-Works** [comment]
Wait. File name changed but no code change?

File: src/social-network-provider/twitter.com/utils/fetch.ts [hunk 1]
```diff
@@ -183,7 +183,7 @@ export const postImageParser = async (node: HTMLElement) => {
                         const content = await Services.Steganography.decodeImageUrl(url, {
                             pass: posterIdentity.toText(),
                         })
-                        return /https:\/\/.+\..+\/%20(.+)%40/.test(content) ? content : ''
+                        return /https:\/\/.+\..+\/(\?PostData_v\d=)?%20(.+)%40/.test(content) ? content : ''
```
Comment (Jack-Works) [inline src/social-network-provider/twitter.com/utils/fetch.ts:186]: does this break the backward compatibility?

Comment (guanbinrui) [reply src/social-network-provider/twitter.com/utils/fetch.ts:186]: I have been tested on old posts. They are working fine.

**Pull request merged**